reference/proto-definition.md

```diff
@@ -73,7 +73,8 @@ PROTO Bicycle [
 
 IS statements may appear inside the PROTO definition wherever fields may appear.
 IS statements shall refer to fields defined in the PROTO declaration.
-Multiple IS statements for the same field in the PROTO interface declaration is valid.
+Multiple IS statements for the same field in the PROTO interface declaration is valid but it is discouraged to use multiple times fields containing [Solid](solid.md) or [Joint](joint.md) nodes.
```

Comment:
Slight syntax problem - "multiple times" should come at the end of the sentence, after "nodes"